### PR TITLE
5.1.4 - release notes

### PR DESCRIPTION
**Pull request by promanenko**

### CODE REVIEW

File: docs/modules/release-notes/partials/nav.adoc [hunk 1]
```diff
@@ -1,3 +1,4 @@
+** xref:release-notes:5-1-3.adoc[5.1.4]
```
Comment (zhmaeff) [inline docs/modules/release-notes/partials/nav.adoc:1]:
```suggestion
** xref:release-notes:5-1-4.adoc[5.1.4]
```

File: docs/modules/release-notes/pages/5-1-4.adoc [hunk 2]
```diff
@@ -0,0 +1,16 @@
+= 5.1.4 Release Notes
+:description: These release notes list any new features, enhancements, and fixes that were made between version 5.1.3 and 5.1.4 of Hazelcast Management Center (MC).
+
+{description}
+
+[[enh-514]]
+== Enhancements
+
+* Jet metrics storage stores only Jet-related metrics. [MC-1532]
```
Comment (JakeSCahill) [inline docs/modules/release-notes/pages/5-1-4.adoc:9]: What was the previous behavior? How does this enhancement benefit the user?

File: docs/modules/release-notes/pages/5-1-4.adoc [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+[[fixes-514]]
+== Fixes
+
+* Fixed the disabled "Upgrade Cluster" button after successfull rolling upgrade. [MC-1569]
```
Comment (JakeSCahill) [inline docs/modules/release-notes/pages/5-1-4.adoc:14]:
```suggestion
* The *Upgrade Cluster* button is no longer disabled after you upgrade your members' codebase version during a rolling upgrade. [MC-1569]
```

File: docs/modules/release-notes/pages/5-1-4.adoc [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+* Fixed an exception on `hz-mc start`. [MC-1403]
```
Comment (JakeSCahill) [inline docs/modules/release-notes/pages/5-1-4.adoc:15]:
```suggestion
* The `hz-mc start`command no longer throws ClassNotFoundException. [MC-1403]
```

File: docs/modules/release-notes/pages/5-1-4.adoc [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+* Fixed various Hazelcast Cloud integration issues. [MC-1591]
```
Comment (JakeSCahill) [inline docs/modules/release-notes/pages/5-1-4.adoc:16]: Can we be more specific? It looks like the work involved a lot of work to do with Pendo integration. How does this fix affect the user?

**Review by JakeSCahill — CHANGES_REQUESTED** [review]
The enhancement sounds like it was a bug. We should clarify how this helps users.

I also added a few suggestions to focus the fix on how it benefits the user.

File: docs/modules/release-notes/pages/5-1-4.adoc (hunk 5 above)
Comment (promanenko) [reply docs/modules/release-notes/pages/5-1-4.adoc:16]: it doesn't affect the end user at all. It's just an internal Hazelcast Clout to MC integration.

File: docs/modules/release-notes/pages/5-1-4.adoc (hunk 2 above)
Comment (promanenko) [reply docs/modules/release-notes/pages/5-1-4.adoc:9]: `Jet metrics storage stores only Jet-related metrics leading to less memory consumption.` - is it OK?

Comment (JakeSCahill) [reply docs/modules/release-notes/pages/5-1-4.adoc:9]: So previously, we were storing unrelated metrics in Jet metrics storage? Isn't that a bug fix?

`We no longer store unrelated metrics with Jet metrics, leading to less memory consumption.`?

Comment (promanenko) [reply docs/modules/release-notes/pages/5-1-4.adoc]: moved to the bugfixes block

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**